src/RA_Core.cpp

```diff
@@ -572,7 +573,7 @@ API HMENU CCONV _RA_CreatePopupMenu()
         AppendMenu(hRA, pConfiguration.IsFeatureEnabled(ra::services::Feature::Hardcore) ? MF_CHECKED : MF_UNCHECKED, IDM_RA_HARDCORE_MODE, TEXT("&Hardcore Mode"));
         AppendMenu(hRA, MF_SEPARATOR, 0U, nullptr);
 
-        AppendMenu(hRA, MF_POPUP, reinterpret_cast<UINT_PTR>(hRA_LB), TEXT("Leaderboards"));
+        GSL_SUPPRESS_TYPE1 AppendMenu(hRA, MF_POPUP, reinterpret_cast<UINT_PTR>(hRA_LB), TEXT("Leaderboards"));
```

Comment:
As far as I'm aware, this reinterpret_cast can't be avoided. Maybe in the future it can be but if it's low-level it can't be helped sometimes. So it's suppressed, there are many cases like this.